src/providers/incoming-data/incoming-data.ts

```diff
@@ -256,6 +252,28 @@ export class IncomingDataProvider {
     else this.goSend(address, amount, message, coin);
   }
 
+  private handleEthereumUri(data: string, redirParams?: RedirParams): void {
+    this.logger.debug('Incoming-data: Ethereum URI');
+    let amountFromRedirParams =
+      redirParams && redirParams.amount ? redirParams.amount : '';
+    const coin = Coin.ETH;
+    const value = /[\?\&]value=(\d+([\,\.]\d+)?)/i;
+    const gasPrice = /[\?\&]gasPrice=(\d+([\,\.]\d+)?)/i;
+    const parsedAmount = value.exec(data)[1];
+    const requiredFeeParam = gasPrice.exec(data)[1];
+    const address = this.bwcProvider
+      .getCore()
+      .Validation.get('ETH')
+      .sanitizeEthereumUri(data);
```

Comment:
Getting error here ```
ERROR in src/providers/incoming-data/incoming-data.ts(267,8): error TS2339: Property 'sanitizeEthereumUri' does not exist on type 'IValidation'. ``` 

Maybe it depends of BWC, right? Could you link to that part of code on bwc?

Reply:
https://github.com/bitpay/bitcore/pull/2421/files#diff-0f910e8ca67a6ad933214c9559bcc118R10